### PR TITLE
`concepts.md` update VTXO + address encoding

### PR DESCRIPTION
**Pull request by louisinger**
Document ark address and VTXO scripts. 
maybe concepts.md is the wrong place ?

@tiero @altafan please review

### CODE REVIEW

**Comment by tiero** [comment]
It should be under Developers, not Learn section

**Comment by altafan** [comment]
The Developers section is meant for client SDKs at the moment. Maybe better to move again under Learn with just a single example of valid VTXO script (we have already one in Learn)?

File: docs/learn/concepts.md [hunk 1]
```diff
@@ -46,15 +46,56 @@ All time periods used on timelocks (**5s**, **24h**, **4w**, **1y**) are arbitra
...
 
-A VTXO should appear on-chain only if the VTXO owner decided to unilaterally exit the Ark. It's locked by a taproot script that has 2 closures
 
-1. **Redeem** lets the user to spend the VTXO unilaterally after a CSV delay. The delay requires the tx to be onchain, so the user can use it only by unrolling a tree. The delay allows the Server to broadcast a forfeit tx if VTXO was already spent off-chain.
-2. **Forfeit** expects both parties (owner and Server) to sign the spending transaction. It is used to forfeit the VTXO off-chain and give it back to the Server when it's spent.
+A VTXO should appear on-chain only if the VTXO owner decided to unilaterally exit the Ark. It's locked by a taproot script that must contain only two types of spending paths:
```
Comment (altafan) [inline docs/learn/concepts.md:50]:
```suggestion
A VTXO should appear on-chain only if the owner decided to unilaterally exit the Ark.

A VTXO is locked by a taproot script that must contain these kinds of spending conditions: 
```

File: docs/learn/concepts.md [hunk 2]
```diff
@@ -46,15 +46,56 @@ All time periods used on timelocks (**5s**, **24h**, **4w**, **1y**) are arbitra
...
 
-1. **Redeem** lets the user to spend the VTXO unilaterally after a CSV delay. The delay requires the tx to be onchain, so the user can use it only by unrolling a tree. The delay allows the Server to broadcast a forfeit tx if VTXO was already spent off-chain.
-2. **Forfeit** expects both parties (owner and Server) to sign the spending transaction. It is used to forfeit the VTXO off-chain and give it back to the Server when it's spent.
+A VTXO should appear on-chain only if the VTXO owner decided to unilaterally exit the Ark. It's locked by a taproot script that must contain only two types of spending paths: 
+
+#### Redeem Paths
+Allow unilateral spending after a CSV delay. Each path requires:
+- The transaction must be on-chain (the VTXO has to be unrolled on-chain)
+- The delay must not be shorter than a threshold set by the Server
+- Only the designated user(s) signature is needed
```
Comment (altafan) [inline docs/learn/concepts.md:58]:
```suggestion
#### Redeem
A redeem closure allows the owner of a VTXO to spend it unilaterally, without the collaboration of the server and must respect the following rules:
- Must be delayed with CSV
- The delay must not be shorter than a threshold set by the server
- Must not include server pubkey/signature
- At least one redeem closure must be included in the VTXO script
```

File: docs/learn/concepts.md [hunk 3]
```diff
@@ -46,15 +46,56 @@ All time periods used on timelocks (**5s**, **24h**, **4w**, **1y**) are arbitra
...
+
+// 2-sig redeem path
+<delay> CHECKSEQUENCEVERIFY DROP <pubkey1> CHECKSIGVERIFY <pubkey2> CHECKSIG
+```
+
+#### Forfeit Paths
+
+Allow cooperative spending between parties. Each path requires:
+- The Server's signature must always be included
+- Can optionally require additional signatures
```
Comment (altafan) [inline docs/learn/concepts.md:74]:
```suggestion
#### Forfeit

A forfeit closure allows the owner of VTXO to spend it off-chain in collaboration with the server and must respect the following rules:
- Must not be delayed
- Must include server's pubkey
- At least one forfeit closure must be included in the VTXO script
```

File: docs/learn/concepts.md [hunk 4]
```diff
@@ -46,15 +46,56 @@ All time periods used on timelocks (**5s**, **24h**, **4w**, **1y**) are arbitra
...
+
+```btcscript
+// 1-sig forfeit path
+<pubkey> CHECKSIGVERIFY <server_pubkey> CHECKSIG
+
+// 2-sig forfeit path
+<pubkey1> CHECKSIGVERIFY <pubkey2> CHECKSIGVERIFY <server_pubkey> CHECKSIG
+```
+
+#### Default VTXO Script
```
Comment (altafan) [inline docs/learn/concepts.md:86]: i would just drop this title

**Pull request closed without merging**